CHANGELOG

```diff
@@ -1,3 +1,10 @@
+0.4.0
+ ===== 
```

Comment:
I'm seeing non-ascii characters sprinkled around this snippet, like before and after the `=` signs.

(If you can't see them, try pasting the string into a Python console as a string, and then `print`ing that string)